hawkmoth/__main__.py

```diff
@@ -30,7 +30,11 @@ def main():
     from FILE, along with the generated C Domain directives, to standard
     output. Include metadata with verbose output.""")
     parser.add_argument('file', metavar='FILE', type=filename, action='store',
-                        help='The C source or header file to parse.')
+                        help='The C/C++ source or header file to parse.')
```

Comment:
Nitpick, I'm slightly averse to using "C/C++" anywhere, and would generally prefer "C or C++" or "C and C++", depending on the case.

Reply:
That's fair and I don't have any strong preference. I'll update it for a final round.

Reply:
Ok, fixed _most_ of these. Left a total of 4, all in the documentation and in similar contexts, in places that somehow felt weird any other way.

You'll let me know if there are still too many :P